### ISSUE TITLE
Documenting way to run pipeline to generate reads in reference peak set

### ISSUE THREAD

**Issue by gbloeb** [databio/pepatac#215]
Documentation here is quite helpful:
[](http://pepatac.databio.org/en/latest/count_table/)

In the section on:
**Count table with a PEPATAC produced consensus peak set**

you recommend rerunning samples with the frip_ref_peaks set based on consensus peak set generated by the pipeline.

Might be useful to be explicit that the PEPATAC_completed.flag needs to be removed for each sample to facilitate this (unless there's another workaround you'd prefer people to use.

Thanks again for the excellent pipeline!

**Comment by jpsmith5** [comment]
Good catch — the doc was implying the per-sample completion flag handling was automatic. Updated `docs/count_table.md` on `dev` to make it explicit: before re-running with `frip_ref_peaks` set, either delete the `<sample>_PEPATAC_completed.flag` files (one-liner `find <output_dir>/results_pipeline -name '*_PEPATAC_completed.flag' -delete`) or pass `--ignore-flags` to `looper run`. Ships in upcoming v0.14.0. Closing.
